examples/step-60/doc/intro.dox

```diff
@@ -0,0 +1,195 @@
+<br>
+
+<i>This program was contributed by Luca Heltai.
+
+<a name="Introduction"></a><h1>Introduction</h1>
+
+
+<a name="DistributedLagrangemultipliers"></a><h3> Distributed Lagrange multipliers</h3>
+
+
+In this tutorial we consider the case of two domains, $\Omega$ in $R^{\text{spacedim}}$
+and $\Gamma$ in $R^{\text{dim}}$, where $\Gamma$ is embedded in $\Omega$ and thus
+$\Gamma \subseteq \Omega$. We want to solve a partial differential equation on $\Omega$,
+enforcing some conditions on
+the solution of the problem *on the embedded domain* $\Gamma$.
+
+There are two interesting scenarios:
+
+- the geometrical dimension `dim` of the embedded domain $\Gamma$ is the
+same of the domain $\Omega$ (`spacedim`), that is, the spacedim-dimensional measure
+of $\Gamma$ is not zero, or
+
+- the embedded domain $\Gamma$ has an intrinsic dimension `dim` which is smaller
+than that of $\Omega$ (`spacedim`), thus its spacedim-dimensional measure is
+zero; for example it is a curve embedded in a two
+dimensional domain, or a surface embedded in a three-dimensional domain.
+
+In both cases define the restriction operator $\gamma$ as the operator
+that, given a continuous function on $\Omega$, returns its (continuous)
+restriction on $\Gamma$, i.e.,
+\f[
+\gamma : C^0(\Omega) \mapsto C^0(\Gamma), \quad \text{ s.t. } \gamma u = u|_{\Gamma} \in C^0(\Gamma),
+\quad \forall u \in C^0(\Omega).
+\f]
+
+It is well known that the operator $\gamma$ can be extended to a continuous
+operator on $H^1(\Omega)$, mapping functions in $H^1(\Omega)$ to functions in
+$H^1(\Gamma)$ when the intrinsic dimension of $\Gamma$ is the same of $\Omega$,
+and that the same is true, with a less regular range space (namely
```

Comment:
Can you find a way to split this sentence in two (maybe around "that the same is true")? I found it a bit long and so hard to follow.